openlibrary/macros/ReadingLogDropper.html

```diff
@@ -84,11 +84,12 @@
             <div class="reading-lists">
               <p class="reading-list-title">$_('My Reading Lists:')</p>
               <div class="my-lists">
-                $for list in lists:
-                    $if not list['active']:
-                      <p class="list">
-                          <a href="$list.key" class="add-to-list" data-list-cover-url="$list.cover_url" data-list-key="$list.key">$list.name</a>
-                      </p>
+                $if async_load:
+                  $ attrs = 'data-edition-key=%s' % edition_key if edition_key else ''
+                  $ attrs = '%s data-work-key=%s' % (attrs, work_key) if work_key else attrs
+                  <div class="list-loading-indicator" $attrs>Loading<span class="loading-ellipsis">...</span></div>
```

Comment:
SUGGESTION: I wonder if this is more clear than overloading `attrs`
```suggestion
                  $ ed_attr = 'data-edition-key=%s' % edition_key if edition_key else ''
                  $ work_attr = 'data-work-key=%s' % work_key if work_key else ''
                  <div class="list-loading-indicator" $ed_attr $work_attr>Loading<span class="loading-ellipsis">...</span></div>
```